GVFS/GVFS.Virtualization/Background/FileSystemTaskQueue.cs

```diff
@@ -22,6 +22,18 @@ private FileSystemTaskQueue(ITracer tracer, PhysicalFileSystem fileSystem, strin
         {
         }
 
+        public bool IsEmpty
+        {
+            get { return this.data.IsEmpty; }
+        }
+
+        /// <summary>
+        /// Gets the count of items in the queue
+        /// </summary>
+        /// <remarks>
+        /// This is an expensive call on .net core and you should avoid calling
+        /// in performance critical paths.  Use the IsEmpty property instead.

```

Comment:
nit: `Use the IsEmpty property instead` - obviously this suggestion is only valid when you want to know if the queue is empty, and is not a replacement for all usages of `Count` :)